### PR TITLE
[SMALLFIX] Update project description in pom

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <version>1.1.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Alluxio Parent</name>
-  <description>Parent POM of Alluxio project: a Reliable Memory Centric Distributed Storage System</description>
+  <description>Parent POM of Alluxio project: a memory speed virtual distributed storage system</description>
```
Comment (calvinjia) [inline pom.xml]: Should the capitalization be consistent?

Comment (apc999) [inline pom.xml]: also: "memory-speed" instead "memory speed"?

**Comment by uronce-cc** [comment]
@calvinjia @apc999 Thanks, updated.

**Comment by calvinjia** [comment]
LGTM thanks

**Pull request merged**